OxfordBodleian/Ullendorff/BDLeu5.xml

```diff
@@ -241,16 +241,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
 
                         <handDesc>
                             <handNote xml:id="h1" script="Ethiopic">
-                                <seg type="ink">Black, red</seg>
-                                <!-- Add dot -->
-                                <date notBefore="1700" notAfter="1750"></date>
-                                <!-- Add date range in letters -->
+                                <seg type="ink">Black, red.</seg>
+                                <date notBefore="1700" notAfter="1750">First half of the eighteenth century.</date>
```

Comment:
Isn't the date given before information on the ink?

Reply:
You are right Eliana, my mistake. Thank you!